metaworld/envs/mujoco/sawyer_xyz/sawyer_assembly_peg_v2.py

```diff
@@ -0,0 +1,201 @@
+import numpy as np
```

Comment:
perhaps mark this file as moved (git mv) and then apply your changes? i think git should show a diff then.

Reply:
i think these changes need to appear in 2 different commits for this to work, btw